modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/platform.bndrun

```diff
@@ -0,0 +1,50 @@
+##
```

Comment:
@rotty3000 FYI: this file is going into workspace template now.  Thanks for the file.  Let us know if you have any suggested changes.